### PR TITLE
Update Leaflet to 1.6.0

### PR DESCRIPTION
**Pull request by Athemis**
Hi,

This pull request updates Leaflet to its latest version. [Changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md) does not suggest any breaking changes since version 1.3.x but mentions numerous bug fixes.

Did not notice any rendering issues with the updated Leaflet version.

Thanks for considering this request!

### CODE REVIEW

**Comment by Athemis** [comment]
For a live version with updated leaflet see here: https://mc.athemis.de/map

**Comment by fjeddy** [comment]
Be warned, upgrading Leaflet introduces a bug where tiles have a 1px border around them, as his map will show. There are some quick fixes involving css and tiles that might overlap online, but no real viable solution.

This bug exist on earlier versions of Leaflet as well, but it's more or less random, with the latest version it's consistent.

**Comment by elamperti** [comment]
@fjeddy thanks for the heads up! With Firefox 76 I'm just seeing the grid when zooming in or out.
Which browser are you using?

**Comment by fjeddy** [comment]
Chrome, latest version!

**Comment by fjeddy** [comment]
It is a known bug btw, you can read more about it at https://github.com/Leaflet/Leaflet/issues/6101, the issue revolves around Chrome, but the latest version of leaflet enhances it.

I didn't realize Mapcrafter had updated to 1.x already, I was under the impression Mapcrafter still used 0.X, have I missed or misunderstood something, or was it "upgraded" within the past 2 years? The problem is enhanced by using Leaflet 1.X, using 1.3 (current) or 1.6 (suggested update) won't change anything.

**Comment by Athemis** [comment]
Only noticed this when zooming in/out (Chrome 83.0.4103.61). Didn't occur to me as a bug since it was also visible with the leaflet version mapcrafter currently ships with ;)

However, while 1.6 won't fix this particular issue (that is present with 1.3.x as well), it provides other fixes.

**Comment by fjeddy** [comment]
@Athemis It isn't a Leaflet bug or a Mapcrafter bug, but a Webkit bug, this issue is present even on Google's own maps.

But last time I was working on Mapcrafter, then I'm 90% sure it was using 0.X of Leaflet, part of the work I did was updating to the latest version of Leaflet, where I then realized that this bug is drastically enhanced by using Leaflet 1.X instead of 0.X

**Comment by Athemis** [comment]
@fjeddy Yeah, I fully understood that. I probably misphrased by last comment to be a bit ambiguous. Let me rephrase it: It didn't occur to me as a bug (regardless of whether it is a browser bug, rendering engine bug, or leaflet bug), because everything looked identical to what I was used to with my current mapcrafter map.
